pkgs/applications/editors/emacs/elisp-packages/manual-packages/cask/package.nix

```diff
@@ -1,30 +1,42 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-, bash
-, emacs
-, python3
+{
+  lib,
+  ansi,
+  coreutils,
+  dash,
+  ecukes,
+  el-mock,
+  ert-async,
+  ert-runner,
+  f,
+  fetchFromGitHub,
+  git,
+  gitUpdater,
+  melpaBuild,
+  noflet,
+  package-build,
+  s,
+  servant,
+  shell-split-string,
 }:
 
-stdenv.mkDerivation (finalAttrs: {
+melpaBuild (finalAttrs: {
   pname = "cask";
-  version = "0.8.8";
+  version = "0.9.0";
 
   src = fetchFromGitHub {
+    name = "cask-source-${finalAttrs.version}";
     owner = "cask";
     repo = "cask";
     rev = "v${finalAttrs.version}";
-    hash = "sha256-TlReq5sLVJj+pXmJSnepKQkNEWVhnh30iq4egM1HJMU=";
+    hash = "sha256-91rJFsp2SLk/JY+v6G5JmXH5bg9QnT+qhI8ccNJlI4A=";
   };
 
-  doCheck = true;
+  patches = [
+    # Uses LISPDIR substitution var
+    ./0000-cask-lispdir.diff
+  ];
 
-  nativeBuildInputs = [ emacs ];
-  buildInputs = [
-    bash
-    python3
-  ]
-  ++ (with emacs.pkgs; [
+  packageRequires = [
```

Comment:
Where do you find these dependencies?  Maybe an update is needed.

It seems these packages are not referenced anywhere:
- dash
- ecukes
- el-mock
- ert-async
- ert-runner
- noflet
- servant
- shell-split-string

While, these referenced packages are not listed here:
- commander
- epl
- shut-up

Reply:
Afaicr I take them from (older?) Cask file: https://github.com/cask/cask/blob/master/Cask